plugin/src/main/scala/chisel3/internal/plugin/BundleComponent.scala

```diff
@@ -118,17 +115,6 @@ private[plugin] class BundleComponent(val global: Global, arguments: ChiselPlugi
       (primaryConstructor, paramAccessors.toList)
     }
 
-    def warnOnCloneType(body: List[Tree]): Unit = {
-      body.foreach {
-        case d: DefDef if isNullaryMethodNamed("cloneType", d) =>
-          val msg = "It is no longer necessary to implement cloneType. " +
-            "Mix in chisel3.experimental.AutoCloneType to let the compiler plugin generate it. " +
```

Comment:
seems like the backport should change this message if they *do* mix in AutoCloneType?

Reply:
Yeah agreed. We should do that by keeping the trait around but deprecating it as mentioned elsewhere.